components/automate-ui/src/app/components/sidebar/sidebar.component.html

```diff
@@ -6,19 +6,19 @@
     <div class="nav-items">
       <span *ngFor="let menuGroup of menuGroups$ | async">
         <div
-          *ngIf="isVisible(menuGroup.visible) | async"
+          *ngIf="isGroupVisible(menuGroup)"
           class="menu-item-groups"
         >
           <div class="group">{{ menuGroup.name }}</div>
           <span *ngFor="let menuItem of menuGroup.items">
-            <span *ngIf="(isVisible(menuItem.visible) | async) && !menuItem.authorized">
+            <span *ngIf="menuItem.visible && !menuItem.authorized">
```

Comment:
why remove `async` here?

Reply:
@bcmdarroch This was a back and forth refactor between static and observable values, which is added back.